### DISCUSSION TITLE
请教一下生成vnote生成HTML或PDF的思路

### DISCUSSION THREAD

**Discussion by w2534073922** [tamlok/vnote#2562] [General]
工作中有大量的markdown需要转换成HTML/PDF，但是markdown内容中掺杂了很多HTML标签和style标签，用了很多软件或脚本都不能完美的转换，基本上都是这些问题：
- 样式过于简陋，只是对markdown进行了最基本的HTML转换
- 转换后原有的style标签变成了纯文本，内联样式失效
- 无法进行脚本化

只有vnote的转换效果比较好但是无法进行脚本化，请问解决这个问题有什么思路吗，vnote是如何实现生成HTML/PDF的呢

**Comment by tamlok** [comment]
脚本化是一个可以考虑的特性，但目前暂时没有时间。

Pandoc转换的效果看过了吗？
